lib/tasks/routes.rake

```diff
@@ -3,11 +3,19 @@ unless defined?(Rails)
   task routes: :environment do
     require './config/application.rb'
     ApplicationEndpoint.routes.each do |api|
-      version = api.route_version.ljust(20)
-      method = api.route_method.ljust(10)
-      path = api.route_path.ljust(40)
-      description = api.route_description
-      puts "#{version} #{method} #{path} # #{description}"
+      if api.respond_to? :version
+        version     = api.version.to_s.ljust(20)
+        method      = api.request_method.ljust(10)
+        path        = api.path.ljust(40)
+        description = api.description
+        puts "#{version} #{method} #{path} # #{description}"
+      else
+        version     = api.route_version.to_s.ljust(20)
+        method      = api.route_method.ljust(10)
+        path        = api.route_path.ljust(40)
+        description = api.route_description
+        puts "#{version} #{method} #{path} # #{description}"
```

Comment:
@aemadrid would you mind moving this print out of the conditional? If you move it after the if/else, it will make [rubocop](https://github.com/bbatsov/rubocop), the static code analyzer, happier.

Thanks for the PR!